src/frontend/src/app/pages/proposal-details/closed-proposal-summary/closed-proposal-summary.component.ts

```diff
@@ -0,0 +1,334 @@
+              <app-value-col [attr.labelledby]="'reviewed-commits-' + i">
+                {{ review.reviewCommits.length }} out of
+                {{ commitList().length }}
+              </app-value-col>
+
+              <app-key-col [id]="'review-link-id-' + i">
+                Full review
+              </app-key-col>
+              <app-value-col [attr.labelledby]="'review-link-' + i">
+                Review Link
+              </app-value-col>
+            </app-key-value-grid>
+          </div>
+        </cg-card>
+      }
+
+      <h2 class="h4">Commits</h2>
+      @for (commit of commitList(); track commit.commitId; let i = $index) {
+        <cg-card class="commit">
+          <div slot="cardContent">
+            <app-key-value-grid [columnNumber]="1">
+              <app-key-col [id]="'commit-id-' + i">ID</app-key-col>
+              <app-value-col [attr.labelledby]="'commit-id-' + i">
+                <a
+                  class="commit__link"
+                  [href]="
+                    'https://github.com/dfinity/ic/commit/' + commit.commitId
+                  "
+                  target="_blank"
+                  rel="nofollow noreferrer"
+                >
+                  {{ commit.commitId }}
+                </a>
+              </app-value-col>
+
+              <app-key-col [id]="'reviewed-by-id-' + i">
+                Reviewed by
+              </app-key-col>
+              <app-value-col [attr.labelledby]="'reviewed-by-id-' + i">
+                {{ commit.reviewedCount }} out of
+                {{ reviewList.length }} reviewers
+              </app-value-col>
+
+              <app-key-col [id]="'matches-descr-id-' + i">
+                Matches description
+              </app-key-col>
+              <app-value-col [attr.labelledby]="'matches-descr-id-' + i">
+                Yes ({{ commit.matchesDescriptionCount }}) No ({{
+                  commit.reviewedCount - commit.matchesDescriptionCount
+                }})
+              </app-value-col>
+            </app-key-value-grid>
+
+            <div class="commit__highlights">
+              <div class="commit__highlights-label">Reviewer highlights</div>
+              <ul>
+                @for (
+                  highlight of commit.highlights;
+                  track highlight.reviewerId
+                ) {
+                  <li class="commit__highlights-content">
+                    Reviewer #{{ highlight.reviewerId }} said:
+                    <span class="commit__highlights-quote">
+                      "{{ highlight.text }}"
+                    </span>
+                  </li>
+                }
+              </ul>
+            </div>
+          </div>
+        </cg-card>
+      }
+    }
+  `,
+})
+export class ClosedProposalSummaryComponent implements OnInit {
+  public readonly proposal = input.required<Proposal>();
+
+  public readonly reviewList = toSignal(this.reviewService.reviewList$);
+
+  public votesAdopt = computed(() => {
```

Comment:
I didn't think of this while we were talking about it last time, but to avoid iterating over `reviewList` multiple times, we could have a single signal for all of the stats. This could also be done in a more functional way using `Array.reduce()`.

```typescript
public proposalStats = computed(() => this.reviewList()?.reduce((accum, review) => ({
    adopt: review.reviewerVote === 'ADOPT' ? accum.adopt + 1 : accum.adopt,
    reject: review.reviewerVote === 'REJECT' ? accum.reject + 1 : accum.reject,
    buildReproduced: review.buildReproduced ? accum.buildReproduced + 1 : accum.buildReproduced,
  }), { adopt: 0, reject: 0, buildReproduced: 0 }) ?? null
);
```

Commits is a little more complicated, so let's go through it together offline.